docs/index.qmd

```diff
@@ -172,17 +172,18 @@ of course, more detail requires more parameters.
 
 ## Installing the compiled executable
 
-Binaries of `ribasim_cli` can be downloaded from
+Binaries of Ribasim CLI can be downloaded from
```

Comment:
```suggestion
Binaries of Ribasim CLI (command-line interface) can be downloaded from
```

Might be good to introduce the term CLI at first appearance in the docs.

Reply:
Sounds good